README.md

```diff
@@ -155,6 +162,8 @@ For getting started with that, you can look at the `extractActions()` function i
 
 ```bash
 npm install ytdl-core
+
+# Replace "npm install" with "yarn add" when using yarn
```

Comment:
Would prefer subsections for both npm and yarn, instead of a comment

Reply:
The only difference is that single substitution though, the rest is 100% the same. Should I just add a new code block then?